.editorconfig

```diff
@@ -0,0 +1,100 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = false
+max_line_length = 120
+tab_width = 4
+ij_continuation_indent_size = 8
+ij_formatter_off_tag = @formatter:off
+ij_formatter_on_tag = @formatter:on
+ij_formatter_tags_enabled = false
+ij_smart_tabs = false
+ij_visual_guides = none
+ij_wrap_on_typing = false
+
+[{*.kt,*.kts}]
```

Comment:
This `.editorconfig` file was generated by IDEA with the default settings.

![image](https://user-images.githubusercontent.com/54322958/141696137-b04c38a9-58fc-4f3f-8ce7-235494578b9a.png)

Reply:
I would suggest reiterating through the file and try to understand how does it work :) Generated file does not help you. For example, if you would take a look at what has been generated @kkarnauk, you would not ask that question: 
> Fair, I agree. I'll fix it. But anyway, why doesn't IDEA add new lines when creating Kotlin files? :(

Reply:
Now I know that it's because of `.editorconfig`. But:
1) Default settings for it is NOT to add new lines for scripts
2) It doesn't add new lines if there's no an `.editorconfig` file anyway. It makes me sad every time :(

Reply:
I've read all the settings and I think that all of them are important in order to maintain the code style of the project.